### DISCUSSION TITLE
How to build the game?

### DISCUSSION THREAD

**Discussion by Radicaal13** [FunkinCrew/Funkin#1567] [Compiling help] [answered]
I'm super extremely new to coding. Most I've done is inspect element. But now I'm building the game. I believe I have everything installed, but when trying to run "lime test windows" this error message pops up:
source/TitleState.hx:61: characters 16-24 : Module APIStuff does not define type APIStuff
source/TitleState.hx:61: characters 16-24 : ... For function argument 'api'
C:/Users/(My name)/Downloads/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/Users/My name)/Downloads/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
I have no idea what it means, but I do know that I need help.

**Comment by ghost** [comment]
Make sure source/APIStuff.hx has the exact text stated in the readme

**Comment by soulware1** [comment]
YOU NEED 2000 BRICKS

**Comment by NotHyper-474** [comment]
See this https://github.com/ninjamuffin99/Funkin#ignored-files

**Comment by TheHENOOB** [comment, marked as answer]
Optional: The Deprecated Warnings are shown when a certain function on Haxe was renamed and later will be removed, You can fix it by changing the names of the functions in the scripts, but even if you compile FNF with those warnings the game will run normally, Just a little info

**Comment by ghost** [comment]
> Optional: The Deprecated Warnings are shown when a certain function on Haxe was renamed and later will be removed, You can fix it by changing the names of the functions in the scripts, but even if you compile FNF with those warnings the game will run normally, Just a little info

Some functions have different cases of use that use different formatting. It would take some work to prevent deprecation warnings.
